data/jsons/dependencies.json

```diff
@@ -126,12 +126,14 @@
 	{
 	    "dependency": "chrpath",
 	    "type": "exec",
-	    "exec": "chrpath"
+	    "exec": "chrpath",
+            "required": "true"
```

Comment:
also, it shouldn't have quotes in "true", just a boolean as supported by JSON.

it would be worth to write a JSON Schema for your dependencies.json, use our common load+check block as done by node-type-gen.py so we can avoid mistakes in there.